tests/rbd_mirror/rbd_mirror_utils.py

```diff
@@ -594,7 +594,7 @@ def create_pool(self, **kw):
             self.exec_cmd(cmd="rbd pool init {}".format(kw.get("poolname")))
 
     def create_image(self, **kw):
-        cmd = "rbd create -s {} {} --image-feature exclusive-lock,journaling".format(
+        cmd = "rbd create -s {} {} --image-feature exclusive-lock,journaling,layering".format(
```

Comment:
Can we please modify this method to create image with default features (layering would be enabled by default if we don't override) and then enable journaling feature using feature enable command ?

Reply:
Done.
Created new method to add required image features.